package.json

```
@@ -6,6 +6,7 @@
     "lint": "resin-lint lib test tools",
     "pretest": "npm run lint",
     "test": "mocha --compilers coffee:coffee-script/register test",
+    "test:local": "env LOCAL_TESTS=true npm test",
```

Why the `env`? I've always seen it written as
```suggestion
    "test:local": "LOCAL_TESTS=true npm test",
```

---

I've read that is a best practice to ensure portability... it works both ways though